JitsiConference.js

```diff
@@ -2530,6 +2553,57 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
 
 };
 
+/**
+ * Called when the chat room reads a new list of properties from jicofo's
+ * presence. The properties may have changed, but they don't have to.
+ *
+ * @param {Object} properties - The properties keyed by the property name
+ * ('key').
+ * @private
+ */
+JitsiConference.prototype._propertiesChanged = function(properties) {
+    const changed = isEqual(properties, this.properties);
+
+    this.properties = properties;
+    if (changed) {
+        this.eventEmitter.emit(JitsiConferenceEvents.PROPERTIES_CHANGED);
+
+        // Some of the properties need to be added to analytics events.
+        const analyticsKeys = [
+
+            // The number of jitsi-videobridge instances currently used for the
+            // conference.
+            'bridge-count',
+
+            // The conference creation time (set by jicofo).
+            'created-ms',
+            'octo-enabled'
+        ];
+
+        analyticsKeys.forEach(key => {
+            if (properties[key] && properties[key].value !== undefined) {
```

Comment:
Can a property go from defined to undefined and therefore be interesting to log?

Reply:
That's a good point. I can't think of why any of the existing ones would go to undefined, and I'd have to add the code to remove a "permanent property". So I'll keep it like this (which is the same as the current behaviour)